config/routes.rb

```diff
@@ -453,7 +453,9 @@
       resource :aggregation, controller: "/admin/settings/aggregation_settings", only: %i[show update]
       resource :mail_notifications, controller: "/admin/settings/mail_notifications_settings", only: %i[show update]
       resource :api, controller: "/admin/settings/api_settings", only: %i[show update]
-      resource :work_packages, controller: "/admin/settings/work_packages_settings", only: %i[show update]
+      # It is important to have this named something else than "work_packages".
+      # Otherwise the angular ui-router will also recognize that as a WorkPackage page and apply according classes.
+      resource :work_package_tracking, controller: "/admin/settings/work_packages_settings", only: %i[show update]
```

Comment:
🟢 the resulting route `/admin/settings/work_package_tracking` makes sense.

but I cannot help to wonder, why all the other entries of the menu do not match a common routing pattern:
- Settings => `/admin/settings/work_package_tracking`
- Types => `/types`
- Status => `/statuses`
- Workflows => `/workflows/edit`
- Custom actions => `/admin/custom_actions`

Reply:
I mean, this is not part of a bugfixing scope, but should we monitor and eventually fix that? or is this only for me disturbing? :D

Reply:
Well, it is none for this bugfix. Let's skip this here and see, if it reappears as an issue eventually.